doc/releases/migration-guide-3.5.rst

```diff
@@ -277,7 +277,7 @@ Power Management
 Bootloader
 ==========
 
-* The :kconfig:option:`CONFIG_BOOTLOADER_SRAM_SIZE` default value is now ``0`` (was
+* The ``CONFIG_BOOTLOADER_SRAM_SIZE`` default value is now ``0`` (was
```

Comment:
It's ok to not change this, this will still render as a code literal, and it's ok to keep the semantic of this having once been an actual/existing Kconfig option.
To put it differently, there isn't today, nor I don't think I will ever add in the future, a check on whether `:kconfig:option:`s actually exist

Reply:
I thought it threw errors when I left them before

Reply:
You have me confused for a bit so I just double-checked :) Can confirm there is *no* warning in case of dangling references for that particular Sphinx role (unlike e.g. `:ref:` or `:zephyr:code-sample:`)